src/index.js

```diff
@@ -0,0 +1,216 @@
+    eslint: {
+      plugins: ['eslint-plugin-react', 'eslint-plugin-prettier'],
+      baseConfig: {
+        extends: [
+          'plugin:react/recommended',
+          'eslint-config-prettier',
+          'eslint-config-prettier/react'
+        ]
+      },
+      rules: {
+        // There are external dependencies we sometimes can't control, and
+        // renaming their variable to lowercase would not be idiomatic
+        'babel/new-cap': 'off',
+        // Allow using class methods with static/non-instance functionality
+        // React lifecycle methods commonly do not use an instance context for anything
+        'class-methods-use-this': 'off',
+        // We turn off consistent-return to allow more efficient early returning from functions
+        'consistent-return': 'off',
+        // Allow console during development, otherwise throw an error
+        'no-console': process.env.NODE_ENV === 'development' ? 'off' : 'error',
+        // Shadowing variables can be useful with array reduction and other FP calls
+        'no-shadow': 'off',
+        // Add extra lines between certain statements to improve readability and breathe-ability via whitespace
+        'padding-line-between-statements': [
+          'error',
+          { blankLine: 'always', prev: ['const', 'let', 'var'], next: '*' },
+          {
+            blankLine: 'never',
+            prev: ['const', 'let', 'var'],
+            next: ['const', 'let', 'var']
+          },
+          { blankLine: 'always', prev: 'multiline-block-like', next: '*' },
+          {
+            blankLine: 'always',
+            prev: '*',
+            next: ['if', 'do', 'for', 'switch', 'try', 'while']
+          },
+          { blankLine: 'always', prev: '*', next: 'return' }
+        ],
+        // Our frontend strives to adopt functional programming practices, so we prefer const over let
+        'prefer-const': 'error',
+        // Use prettier to automatically format the style of application code
+        'prettier/prettier': [
+          'error',
+          {
+            singleQuote: true,
+            trailingComma: 'none',
+            bracketSpacing: true,
+            jsxBracketSameLine: true
+          }
+        ],
+        // Validate JSX has key prop when in array or iterator
+        'react/jsx-key': 'error',
+        // Prevent comments from being inserted as text nodes
+        'react/jsx-no-comment-textnodes': 'error',
+        // Prevent usage of unsafe target="_blank", ensure anchors also have rel="noreferrer noopener"
+        'react/jsx-no-target-blank': 'error',
+        // Ensure JSX components are PascalCase
+        'react/jsx-pascal-case': 'error',
+        // Disable forced validation of React prop types
+        'react/prop-types': 'off'
+      }
+    }
+  });
+
+  if (options.eslint) {
+    neutrino.use(loaderMerge('lint', 'eslint'), options.eslint);
+  }
+
+  if (options.cssModules) {
+    neutrino.config.module
+      .rule('plain-style')
```

Comment:
What's the difference between this and the stock Neutrino behaviour? Could you document it in a comment?
Should it be upstreamed to neutrino-middleware-style-loader?